### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by YlanGarnier**
# Description

Updated Readme to be up to date with S1 2023 working team
Related to #1

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to an issue
- [x] I have linked this PR to a milestone
- [x] I have linked this PR to a project
- [x] I have updated the README and other relevant documents (guides...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ If you're interested in how the project is organized at a higher level, please c
 ## Our PoC team ❤️
 
 Developers
-| [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy) | [<img src="https://github.com/MrZalTy.png?size=85" width=85><br><sub>[Developer's name]</sub>](https://github.com/MrZalTy)
+| [<img src="https://github.com/YlanGarnier.png?size=85" width=85><br><sub>[Ylan GARNIER]</sub>](https://github.com/YlanGarnier) | [<img src="https://github.com/paulogarithm.png?size=85" width=85><br><sub>[Paul PARISOT]</sub>](https://github.com/paulogarithm) | [<img src="https://github.com/Saverio976.png?size=85" width=85><br><sub>[Xavier MITAULT]</sub>](https://github.com/Saverio976)
 | :---: | :---: | :---: |
 
 Manager
-| [<img src="https://github.com/adrienfort.png?size=85" width=85><br><sub>[Manager's name]</sub>](https://github.com/adrienfort)
```
Comment (RezaRahemtola) [inline README.md]: You can remove the brackets around the names:\
`Ylan GARNIER` instead of `[Ylan GARNIER]` :wink:

**Review by RezaRahemtola — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (YlanGarnier) [reply README.md]: done :slightly_smiling_face:

**Review by RezaRahemtola — APPROVED** [review]
LGTM !

**Comment by RezaRahemtola** [comment]
Just make sure to use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) next time @YlanGarnier :wink:

**Pull request merged**